package.json

```diff
@@ -88,7 +85,12 @@
         "D2.updateOnSave": {
           "type": "boolean",
           "default": false,
-          "description": "Automatically update on saving the d2 file."
+          "description": "Automatically update on saving the D2 file."
+        },
+        "D2.formatOnSave": {
+          "type": "boolean",
+          "default": false,
+          "description": "Automatically format on saving the D2 file."
```

Comment:
I noticed it would automatically format on save based on the `Editor: Format On Save` option enabled, even with this set to false. I think we can probably remove this setting since the editor one already handles it. Alternatively we could make this selection from 3 options (default, no, yes) and only override the editor setting if no or yes is selected.

Reply:
I thought about this, but stopped before coming up with a solution :) This is probably a good idea.

Reply:
I tried the three options and six combinations settings you get with the editor Format on Save and then default/Always/Never, there is only enough information to get five of the six.  So I took the easy way out and it's all stock VsCode behavior now.